murfi-rt-PyProject/scripts/feedback.sh

```diff
@@ -519,4 +519,58 @@ then
     rm -f $subj_dir/rest/*bold.nii.gz
     rm -f $subj_dir/rest/*bold_mcflirt.nii.gz
     rm -f $subj_dir/rest/*bold_mcflirt_masked.nii.gz
+fi
+
+
+# ONLY if regular mask generation isn't possible
+# As a backup, we can register the template masks to 2vol space
+if [ ${step} = backup_reg_mni_masks_to_2vol ]
+then
+    clear
+    mni_template=MNI152_T1_2mm_LPS_brain.nii.gz
+    dmn_mni=DMNax_brainmaskero2_lps.nii.gz
+    cen_mni=CENa_brainmaskero2_lps.nii.gz
+
+    two_vol_ref=$(ls -t $subj_dir/xfm/series*.nii | head -n1)
+    two_vol_ref="${two_vol_ref::-4}"
+
+    two_vol_ref_bet=${subj_dir}/xfm/two_vol_ref_bet.nii.gz
+    two_vol_ref2mni=${subj_dir}/xfm/two_vol_ref2mni.nii.gz
+    two_vol_ref2mni_mat=${subj_dir}/xfm/two_vol_ref2mni.mat
+    mni2_two_vol_ref_mat=${subj_dir}/xfm/mni2_two_vol_ref.mat
+
+    study_ref=${subj_dir}/xfm/study_ref.nii
+    # if localizer_ref images doesn't exist yet, make it
+    if [ ! -f ${subj_dir}/xfm/localizer_ref.nii ]
+    then
+        mv ${study_ref} ${subj_dir}/xfm/localizer_ref.nii
+    fi
+    echo "Registering masks to reference image from most recent series: ${two_vol_ref}"
```

Comment:
Is `two_vol_ref` the most recent, or first series?

Reply:
most recent! we use the 2-volume scan as a template for the current functional space to register the masks to prior to feedback